tests/test_cli.py

```diff
@@ -225,6 +236,45 @@ def test_auth_handler_cookie():
     assert resp.headers["Cookie"] == token
 
 
+def test_auth_handler_cookie_explicit_token_string():
+    req = WebTestRequest({})
+    token = str(uuid.uuid4())
+    auth = CookieAuthHandler(token=token)
+    with mock.patch("requests.Session.request") as mock_request:
+        resp = auth(req)  # type: ignore
+        mock_request.assert_not_called()
+    assert "Authorization" not in resp.headers
+    assert "Cookie" in resp.headers and len(resp.headers["Cookie"])
+    assert resp.headers["Cookie"] == token
```

Comment:
Could be nice to have a test that does both variant (prefilled token vs retrieved one from request) to validate that either branch results in the same `Cookie` header after parsing (see other PR comments).